variables.tf

```diff
@@ -17,40 +17,35 @@ variable "infrastructure-support" {
   description = "The team responsible for managing the infrastructure. Should be of the form <team-name> (<team-email>)"
 }
 
-variable "ec_engine" {
-  description = "The engine which your ElastiCache Cluster will use. You have a choice of either 'redis' or 'memcached'. The default is 'redis'"
-  type        = "string"
-  default     = "redis"
-}
-
 variable "engine_version" {
   description = "The engine version that your ElastiCache Cluster will use. This will differ between the use of 'redis' or 'memcached'. The default is '4.0.10' with redis being the assumed engine."
   type        = "string"
   default     = "4.0.10"
 }
 
+variable "replication_group_description" {
+  description = "A short description of the replication group"
+  type        = "string"
+}
+
 variable "parameter_group_name" {
   description = "Name of the parameter group to associate with this cache cluster. Again this will differ between the use of 'redis' or 'memcached' and your engine version. The default is 'default.redis4.0'."
   type        = "string"
   default     = "default.redis4.0"
 }
 
+variable "number_cache_clusters" {
+  description = "The number of cache clusters (primary and replicas) this replication group will have. Default is 2"
```

Comment:
Here it says the defaul is 2, however, the actual value is 3.
Do we even need to setup more than one node by default, given this will be typically used as a cache?

Reply:
Default back to 2 and slice on AZs implemented.